core/src/utils/FilesystemUtils.cpp

```diff
@@ -3,7 +3,7 @@
  * FilesystemUtils.cpp
```

Comment:
why moving files? why not keeping the old qt function and creating a new .cpp and .hpp file for the new one